src/test/java/com/github/awwkoala/selenium/CheckBoxesTests.java

```diff
@@ -0,0 +1,36 @@
+package com.github.awwkoala.selenium;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class CheckBoxesTests {
```

Comment:
I would create some "Base" Test class in order to provide common logic for other/future tests.
In the beginning webdriver instantiation and closing